packages/strip/src/index.js

```diff
@@ -38,21 +38,34 @@ export default function strip(options = {}) {
   const sourceMap = options.sourceMap !== false;
 
   const removeDebuggerStatements = options.debugger !== false;
-  const functions = (options.functions || ['console.*', 'assert.*']).map((keypath) =>
-    keypath.replace(/\./g, '\\.').replace(/\*/g, '\\w+')
-  );
 
+  const functions = options.functions || ['console.*', 'assert.*'];
   const labels = options.labels || [];
 
-  const firstpass = new RegExp(`\\b(?:${functions.join('|')}|debugger)\\b`);
-  const pattern = new RegExp(`^(?:${functions.join('|')})$`);
+  const functionsPatterns = functions.map((f) =>
+    f.replace(/\*/g, '\\w+').replace(/\./g, '\\.\\s*')
+  );
+  const labelsPatterns = labels.map((l) => `${l}\\s*:`);
+
+  const firstPassPatterns = removeDebuggerStatements
+    ? [...functionsPatterns, ...labelsPatterns, 'debugger\\b']
+    : [...functionsPatterns, ...labelsPatterns];
+
+  const functionsRE = new RegExp(`^(?:${functionsPatterns.join('|')})$`);
```

Comment:
while we're in here, please use the naming pattern `reFunctions` and `reFirstpass`

Reply:
done